modules/virt-removing-secret-configmap-service-account-vm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc
+
+[id="virt-removing-secret-configmap-service-account-vm_{context}"]
+
+= Removing a secret, ConfigMap, or service account from a virtual machine
+
+Remove a secret, ConfigMap, or service account from a virtual machine by using the {product-title} web console.
+
+.Prerequisites
+
+* You must have at least one secret, ConfigMap, or service account
+that is attached to a virtual machine.
+
+* The virtual machine must be powered off.
+
+.Procedure
+
+. From the side menu, navigate to the *Virtualization* -> *Virtual Machines* page.
+
+. Select a virtual machine to open its *Virtual Machine Overview* page.
+
+. Click the *Environment* tab.
+
+. Find the item that you want to delete in the list of secrets, ConfigMaps, and service accounts.
```

Comment:
This could be combined with the subsequent step so that it's 'Find the item.. and click the Delete button'